fastapi_endpoint.py

```
@@ -0,0 +1,127 @@
+import shutil
+import logging
+import time
+
+from fastapi import FastAPI, Request
+from webui import run_browser_agent
+from filter_mitm_logs import filter_jsonl_file
+from filter_dom import dom_main
+app = FastAPI()
+
+
+def copy_folder(source_folder, destination_folder):
```

Ruff (0.8.2)

Missing return type annotation for public function `copy_folder`

Add return type annotation: `None`

(ANN201)